langkit/dsl_unparse.py

```diff
@@ -20,6 +22,17 @@
 def fqn(prop):
     return "{}.{}".format(prop.struct.name.camel, prop._original_name.lower)
 
+def print_diagnostic(filename, sloc_range, message):
+    line = sloc_range.start.line
+    column = sloc_range.start.column
+    print('In {}, line {}:'.format(filename, line), file=sys.stderr)
+    with open(filename) as f:
+        # Get the corresponding line in the source file and display it
```

Comment:
You should probably use `langkit.diagnostics` here, rt. reinventing the wheel.

Reply:
Done ;). For info, output looks like:

```
ast.py:9174:13: error: Expected ')', got Indent
9173 |             ).map(
     |                   
```